src/target/cortexm.c

```diff
@@ -673,7 +674,6 @@ bool cortexm_probe(adiv5_access_port_s *ap)
 		 * and have no available designer code elsewhere
 		 */
 		PROBE(sam3x_probe);
-		PROBE(ke04_probe);
```

Comment:
If memory serves, this may still be required from some KE04 parts, so let's not delete this line unless you can be certain it's definitely not required - otherwise we might well find ourselves in a game of cat-and-mouse with people un-breaking their parts and breaking other people's.

Reply:
Good point, I am not certain. Thank you for noting this. Added it back.

Thank you for the quick review as well!